vividus-extension-selenium/src/main/java/org/vividus/bdd/steps/ui/GenericElementSteps.java

```diff
@@ -54,9 +54,15 @@ public GenericElementSteps(IBaseValidations baseValidations, ElementActions elem
     /**
      * Checks whether the context contains exact amount of elements by locator
      * @param locator to locate element
-     * @param comparisonRule The rule to compare values
-     * (<i>Possible values:<b> LESS_THAN, LESS_THAN_OR_EQUAL_TO, GREATER_THAN, GREATER_THAN_OR_EQUAL_TO,
-     * EQUAL_TO</b></i>)
+     * @param comparisonRule The rule to match the variable value. The supported rules:
```

Comment:
I think this statement should be updated for different steps, like:
`The rule to match the variable value` -> `The rule to match the quantity of elements`